### ISSUE TITLE
Request small Privacy selection

### ISSUE THREAD

**Issue by SampeiNihira** [uBlockOrigin/uBOL-issues#6]
![10](https://user-images.githubusercontent.com/44166263/193414797-954cf071-3339-4b05-849c-35b9a6d95481.jpg)

It could be of interest to many users of UBO Lite.
I will insert an image with my ideas but you can decide more.
putting a Ping checkbox doesn't seem interesting to me ,because it can be blocked in Chromium-based browsers with the command-line:

--no-pings

[https://www.ghacks.net/2022/08/27/websites-may-write-to-the-clipboard-in-chrome-without-user-permission/](url)

It has been proposed to put this rule in my filters in UBO:

*##+js(acis, navigator.clipboard)

It seems to work,but I have no idea if it can work in UBO Lite.

I hope I have not disturbed too much.
Thank you for your attention.

**Comment by krystian3w** [comment]
Chrome 107 no longer allow read clipboard without prompting asking to allow permission.

If you found new option to read clipboard in Chromium 107+ the best will be open new issue about lack of fixes.

**Comment by SampeiNihira** [comment]
That is just one option,which I assumed interesting.
If it is not interesting to the developer,he can eliminate it.
I use Edge.
And I often notice that not all the new features in Chromium/Chrome are carried over to Edge.
Example:

https://www.chromium.org/updates/chips/

CHIPS cannot work in Edge due to the lack of a required flags.

**Comment by krystian3w** [comment]
Egde/MS insisted that it be fixed directly in the Chromium code.

So as not to fix it with your own hotifix.

**Comment by SampeiNihira** [comment]
Better for everyone if it will be so.
Do you have any alternatives to propose as a substitute for the third idea?
If you do have it instead of just criticizing expose an alternative.

**Comment by SampeiNihira** [comment]
In Edge 107 the setting:

"disable clipboard writing"

is no longer necessary.

Compared to the default setting I prefer:

![3](https://user-images.githubusercontent.com/44166263/198598718-c8428a05-91a7-4a27-a3f5-bc32a22c175a.jpg)

**Comment by gorhill** [comment]
uBO Lite is meant to be as not technical as possible. _"Blocking CSP reports"_ is quite technical, I don't see this fitting in uBOL; how would you even explain this to a computer illiterate person?

uBO is best for this, as its spectrum is for wider audience, from install-and-forget to those who prefer to control as much as possible everything.

**Comment by SampeiNihira** [comment]
Unfortunately, it will not be possible to use UBO in Edge.
And I set up my Edge to have very good security at the Sandbox/Appcontainer level:

![1](https://user-images.githubusercontent.com/44166263/198842733-d218338f-b67b-4227-b3aa-00d18e197b51.jpg)


This is critical in Windows OS.
These settings cannot be obtained in other Chromium-based browsers.

My PC already has Firefox + UBO installed.
But it is necessary to have a Chromium-based browser installed as well.

I would not wish to give up Edge,for example switch to Brave (with its internal adblocker) as will many users of Chromium-based browsers who consider privacy more important than security.

**Comment by gorhill** [comment]
You have different goal with your personal installation than I do have with uBOL, in such case its best for you to find a solution elsewhere than for me to abandon from my goal with uBOL.

**Comment by Yuki2718** [comment]
Off-topic:
`https://www.wilderssecurity.com/threads/ublock-a-lean-and-fast-blocker.365273/page-274#post-3113350`
> With UBO Lite this is not possible now and will not be possible in the future either because mr.Hill refused my request for a small privacy panel.

@SampeiNihira Beacon API is globally blocked by EP with `$ping,third-party` - limited to 3p due to problems but individual 1p cases has been added. Globally blocking CSP reports by filters is simply not possible, the only way is `*$other` if you don't care possible problems. It's likely you confused somthing about `$csp` and csp report or someone told you misinformation.

**Comment by Yuki2718** [comment]
> How is it then possible to pass this test?

It's 1p ping so add `||apps.armin.dev^$ping`.

**Comment by gorhill** [comment]
> How is it then possible to pass this test?

Use uBO?

**Comment by Yuki2718** [comment]
For CSP report `||apps.armin.dev^$other` or `*$other`. I don't remember details, but IIRC at first `*$ping` was added to EP and then adjusted to 3p due to some reports.

**Comment by SampeiNihira** [comment]
@gorhill

No.
AdGuard MV3 to which I added 2 simple rules.
I would have tried UBO Lite if it was possible to do the same.
Surely I get everything wrong,but I proved that it is possible.

**Comment by krystian3w** [comment]
Custom Rules form ABP/EasyList syntax should be possible inject to file after install unpacked addon form SSD/NVMe or HDD (I not recommend USB 1.1 or 2.0).

Harder will be sync - may needed use WinMerge or Meld.

**Comment by Yuki2718** [comment]
@gorhill I think the only matter here is whether uBOL eventually support costom lists and/or My filters.

**Comment by gorhill** [comment]
> whether uBOL will eventually support custom lists and/or My filters

Custom list probably not. Custom filters I haven't decided yet. As things currently are, these features require that I add as part of the package the whole static filtering engine just for the sake of converting filters to rules, and by extension a lot more of development and maintenance work, so not something I look forward too. As said elsewhere, I burned on uMatrix, I do not want that to happen with uBOL, so to keep it as simple as possible is a way to ensure this.

**Comment by devingoodman** [comment]
>If you do not want to use uBO, it's your choice -- you set yourself constraints of wanting to use MV3 and wanting a blocker that does all that uBO can do, then uBOL is just not for you, and I won't change uBOL's goal because of these personal choices of yours. uBOL is another personal project with a clear goal in mind, which I won't throw out because someone somewhere has very specific unmatched needs.

This is very well said and should probably be an autoreply that one of the uBO volunteers can copy/paste for future uBOL feature requests. It doesn't make sense for people to choose the "lite" version and then request features to make it less lite. Just use uBO. 

@gorhill It's really helpful that you've stated the project goals so succinctly bc it's easy for us to understand why you are making the decisions you do. Personally, I love uBO and wouldn't trade it for anything.

**Comment by SampeiNihira** [comment]
@devingoodman

Are you aware that an MV2 extension like UBO will only be available for Firefox (and maybe Brave)?
In other Chromium-based browsers you will have to use UBO Lite by necessity.

**Comment by Yuki2718** [comment]
Guys, please move to https://github.com/uBlockOrigin/uBlock-issues/discussions/2261 for any more discussion.
